src/lib/checkbox/_checkbox-theme.scss

```diff
@@ -68,16 +68,16 @@
     }
   }
 
-  .md-checkbox:not(.md-checkbox-disabled) {
-    &.md-primary .md-checkbox-ripple .md-ripple-foreground {
+  md-checkbox:not(.md-checkbox-disabled) {
```

Comment:
Good point.  I actually had that change only in this PR for testing purposes. I have another PR open for this. See https://github.com/angular/material2/pull/2857

Also I think we can only use the class here, as soon as Andrew's PR is merged.